src/abstractdataframe/abstractdataframe.jl

```diff
@@ -448,68 +454,80 @@ function StatsBase.describe(io::IO, df::AbstractDataFrame; stats::AbstractVector
     return data
 end
 
-# Define 4 functions for getting summary statistics 
+# Define functions for getting summary statistics 
 # use a dict because we dont know which measures the user wants
 # Outside of the `describe` function due to something with 0.7
-function get_stats(col::AbstractArray{<:Real})
-    sumstats = summarystats(col)
+
+function get_stats(col::AbstractArray{<:Union{Real, Missing}})
+    nomissing = collect(skipmissing(col))
+    sumstats = summarystats(nomissing)
     Dict(
         :mean => sumstats.mean,
-        :std => Compat.std(col, mean = sumstats.mean),
+        :std => Compat.std(nomissing, mean = sumstats.mean),
         :min => sumstats.min,
         :q25 => sumstats.q25,
         :median => sumstats.median,
         :q75 => sumstats.q75,
         :max => sumstats.max,
-        :nmissing => nothing,
+        :nmissing => count(ismissing, col),
         :nunique => nothing,
-        :eltype => eltype(col)
+        :first => first(col),
+        :last => last(col),
+        :eltype => Missings.T(eltype(col))
     )
 end
 
-function get_stats(col::AbstractArray{<:Union{Real, Missing}})
-    nomissing = collect(skipmissing(col))
-    sumstats = summarystats(nomissing)
+function get_stats(col::AbstractArray{<:Real})
+    sumstats = summarystats(col)
     Dict(
         :mean => sumstats.mean,
-        :std => Compat.std(nomissing, mean = sumstats.mean),
+        :std => Compat.std(col, mean = sumstats.mean),
         :min => sumstats.min,
         :q25 => sumstats.q25,
         :median => sumstats.median,
         :q75 => sumstats.q75,
         :max => sumstats.max,
-        :nmissing => count(ismissing, col),
+        :nmissing => nothing,
         :nunique => nothing,
-        :eltype => Missings.T(eltype(col))
+        :first => first(col),
+        :last => last(col),
+        :eltype => eltype(col)
     )
 end
 
 function get_stats(col::AbstractArray{>:Missing})
+    nomissing = collect(skipmissing(col))
+    q = try quantile(nomissing, [.25, .5, .75]) catch [nothing, nothing, nothing] end
     Dict(
-        :mean => nothing,
-        :std => nothing,
-        :min => nothing,
-        :q25 => nothing,
-        :median => nothing,
-        :q75 => nothing,
-        :max => nothing,
+        :mean => try mean(nomissing) catch end,
+        :std => try Compat.std(nomissing) catch end,
+        :min => try minimum(nomissing) catch end,
```

Comment:
Would be nice to use `extrema` to avoid one additional pass over the data. If that fails, we can return `nothing` for both minimum and maximum. Same below.

Reply:
fixed.